test/val/val_instructions_test.cpp

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) 2017 The Khronos Group Inc.
```

Comment:
Prefer Google Inc. here since you wrote this whole thing as work product for Google.
But this is acceptable too.